static/app/views/insights/browser/resources/views/resourcesLandingPage.tsx

```diff
@@ -64,23 +64,25 @@ function ResourcesLandingPage() {
         <Layout.Body>
           <Layout.Main fullWidth>
             <PageAlert />
-            <FilterOptionsContainer columnCount={2}>
-              <PageFilterBar condensed>
-                <ProjectPageFilter />
-                <EnvironmentPageFilter />
-                <DatePageFilter />
-              </PageFilterBar>
-              {hasModuleData && (
-                <DomainSelector
-                  emptyOptionLocation="top"
-                  value={filters[SPAN_DOMAIN] || ''}
-                  additionalQuery={[
-                    ...DEFAULT_RESOURCE_FILTERS,
-                    `${SPAN_OP}:[${DEFAULT_RESOURCE_TYPES.join(',')}]`,
-                  ]}
-                />
-              )}
-            </FilterOptionsContainer>
+            <StyledHeaderContainer>
+              <ToolRibbon>
+                <PageFilterBar condensed>
+                  <ProjectPageFilter />
+                  <EnvironmentPageFilter />
+                  <DatePageFilter />
+                </PageFilterBar>
+                {hasModuleData && (
+                  <DomainSelector
+                    emptyOptionLocation="top"
+                    value={filters[SPAN_DOMAIN] || ''}
+                    additionalQuery={[
+                      ...DEFAULT_RESOURCE_FILTERS,
+                      `${SPAN_OP}:[${DEFAULT_RESOURCE_TYPES.join(',')}]`,
+                    ]}
+                  />
+                )}
+              </ToolRibbon>
+            </StyledHeaderContainer>
```

Comment:
Just a heads up, this will clash with
https://github.com/getsentry/sentry/pull/74653